larq_zoo/__init__.py

```diff
@@ -1,29 +1,7 @@
 from larq_zoo import datasets
-from larq_zoo.binary_alex_net import BinaryAlexNet
-from larq_zoo.birealnet import BiRealNet
 from larq_zoo.data import preprocess_input
-from larq_zoo.densenet import (
-    BinaryDenseNet28,
-    BinaryDenseNet37,
-    BinaryDenseNet37Dilated,
-    BinaryDenseNet45,
-)
-from larq_zoo.dorefanet import DoReFaNet
-from larq_zoo.resnet_e import BinaryResNetE18
 from larq_zoo.utils import decode_predictions
-from larq_zoo.xnornet import XNORNet
+import larq_zoo.literature as literature
+import larq_zoo.sota as sota
 
-__all__ = [
-    "BinaryAlexNet",
-    "BiRealNet",
-    "BinaryResNetE18",
-    "BinaryDenseNet28",
-    "BinaryDenseNet37",
-    "BinaryDenseNet37Dilated",
-    "BinaryDenseNet45",
-    "DoReFaNet",
-    "XNORNet",
-    "datasets",
-    "decode_predictions",
-    "preprocess_input",
-]
+__all__ = ["datasets", "decode_predictions", "preprocess_input", "literature", "sota"]
```

Comment:
I think before we move to v1.0 stable, we should find a better place for `datasets`, `decode_predictions` and `preprocess_input` because they might be different for different models.